gwpy/types/series.py

```diff
@@ -845,8 +851,22 @@ def append(self, other, inplace=True, pad=None, gap=None, resize=True):
                     self.xindex.resize((s[0],), refcheck=False)
                 except ValueError as exc:
                     if 'cannot resize' in str(exc):
-                        self.xindex = self.xindex.copy()
-                        self.xindex.resize((s[0],))
+                        try:
+                            self.xindex = self.xindex.copy()
+                            self.xindex.resize((s[0],))
+                        except ValueError as exc2:
```

Comment:
Does the `.copy()` operation need to be inside the try/except?

```suggestion
                        self.xindex = self.xindex.copy()
                        try:
                            self.xindex.resize((s[0],))
                        except ValueError as exc2:
```